### PR TITLE
Fix xgroupDelConsumer method and add test case

### PR DESCRIPTION
**Pull request by dengliming**
A PR for [https://github.com/xetorthio/jedis/issues/2143](https://github.com/xetorthio/jedis/issues/2143)

### CODE REVIEW

File: src/test/java/redis/clients/jedis/tests/commands/StreamsCommandsTest.java [hunk 1]
```diff
@@ -233,7 +233,9 @@ public void xgroup() {
     
     jedis.xgroupDestroy("xgroup-stream", "consumer-group-name");
 
-    //TODO test xgroupDelConsumer
+    //xgroupDelConsumer
```
Comment (sazzad16) [inline src/test/java/redis/clients/jedis/tests/commands/StreamsCommandsTest.java]: Not required.

**Comment by sazzad16** [comment]
Even though this breaks, but it should be considered as a bugfix.

File: src/test/java/redis/clients/jedis/tests/commands/StreamsCommandsTest.java (hunk 1 above)
Comment (dengliming) [reply src/test/java/redis/clients/jedis/tests/commands/StreamsCommandsTest.java]: > Even though this breaks, but it should be considered as a bugfix.

it is.

Comment (dengliming) [reply src/test/java/redis/clients/jedis/tests/commands/StreamsCommandsTest.java]: I can remove it, but what is your review the standards of your code?I doubt it.

**Comment by dengliming** [comment]
> Even though this breaks, but it should be considered as a bugfix.

It is.

File: src/test/java/redis/clients/jedis/tests/commands/StreamsCommandsTest.java (hunk 1 above)
Comment (sazzad16) [reply src/test/java/redis/clients/jedis/tests/commands/StreamsCommandsTest.java]: > I can remove it, but what is your review the standards of your code?I doubt it.

@dengliming I'm afraid I didn't get you.

However, putting a comment `//method` before calling `method()` is absolutely nonsensical.

Comment (dengliming) [reply src/test/java/redis/clients/jedis/tests/commands/StreamsCommandsTest.java]: Sorry,It's my misunderstanding.but the project is moving too slowly,many of the changes were suspended.

**Comment by dengliming** [comment]
Pinging @gkorland :)

**Comment by dengliming** [comment]
@sazzad16 When these changes will be merged to master, I think this bug has a higher priority.

**Comment by sazzad16** [comment]
@dengliming It shouldn't take long (but it may). We need at least one more collaborator to approve this PR.

File: src/test/java/redis/clients/jedis/tests/commands/StreamsCommandsTest.java [hunk 2]
```diff
@@ -233,7 +233,8 @@ public void xgroup() {
     
     jedis.xgroupDestroy("xgroup-stream", "consumer-group-name");
 
-    //TODO test xgroupDelConsumer
+    Long pendingMessageNum = jedis.xgroupDelConsumer("xgroup-stream", "consumer-group-name1", "myconsumer1");
+    assertNotNull(pendingMessageNum);
```
Comment (gkorland) [inline src/test/java/redis/clients/jedis/tests/commands/StreamsCommandsTest.java]:
```
    assertEquals(0L, pendingMessageNum.longValue());
```

**Comment by momomo** [comment]
Wow, this is a breaking change but I am not against it. It has been four months however, and the current maven version still used String.

**Comment by dengliming** [comment]
@momomo [https://github.com/xetorthio/jedis/releases/tag/jedis-3.3.0](https://github.com/xetorthio/jedis/releases/tag/jedis-3.3.0)
```
<dependency>
    <groupId>redis.clients</groupId>
    <artifactId>jedis</artifactId>
    <version>3.3.0</version>
</dependency>
```

**Comment by momomo** [comment]
Maybe update: 

image

I am using 3.2.0 it seems.

**Comment by dengliming** [comment]
@momomo already submit a pr for this. [https://github.com/xetorthio/jedis/pull/2240](https://github.com/xetorthio/jedis/pull/2240)

**Pull request merged**